src/org/traccar/model/Position.java

```diff
@@ -39,6 +39,8 @@
     public static final String KEY_LAC = "lac";
     public static final String KEY_CID = "cid";
     public static final String KEY_FUEL = "fuel";
+    public static final String KEY_FUEL_CONSUMPTION = "fuelConsumption";
+    public static final String KEY_FUEL_INSTANT_CONSUMPTION = "fuelInstantConsumption";
```

Comment:
It is consumption at the time calculated by ECU and accessible via OBD. Frankly, it is useless parameter even for engine diagnostic.

But why not to store it if it is exist. There are a lot of useless attributes we store in different protocols. Or we are trying to avoid it?